internal/reporter.go

```diff
@@ -28,14 +26,6 @@ func NewReporter(server string, tokenService auth.Service, client *http.Client)
 	}
 }
 
-func NewClient(timeout time.Duration, tlsConfig *tls.Config) *http.Client {
```

Comment:
I almost want to keep some flavor of this and then add a deprecation warning. The new changes make it impossible to keep previous functionality working without changes to client code. Maybe that is ok. Shrug.

Reply:
n/a